air/src/execution_step/errors/catchable_errors.rs

```diff
@@ -35,6 +35,14 @@ use std::rc::Rc;
 #[derive(ThisError, EnumDiscriminants, Debug)]
 #[strum_discriminants(derive(EnumIter))]
 pub enum CatchableError {
+    /// This error type is produced by a match to notify xor that compared values aren't equal.
+    #[error("match is used without corresponding xor")]
+    MatchValuesNotEqual,
+
+    /// This error type is produced by a mismatch to notify xor that compared values aren't equal.
+    #[error("mismatch is used without corresponding xor")]
+    MismatchValuesEqual,
+
```

Comment:
I just thought with a fresh view that it's better to place them after `LocalServiceError`, because the latter is more important error type.